crates/bitwarden-core/src/key_management/crypto.rs

```diff
@@ -1412,9 +1400,12 @@ mod tests {
                     iterations: 100_000.try_into().unwrap(),
                 },
                 email: "test@bitwarden.com".into(),
-                private_key: enrollment_response.private_key,
-                signing_key: Some(enrollment_response.signing_key),
-                security_state: Some(enrollment_response.security_state),
+                account_cryptographic_state: WrappedUserAccountCryptographicState::V2 {
+                    private_key: enrollment_response.private_key.clone(),
+                    signing_key: enrollment_response.signing_key.clone(),
+                    security_state: enrollment_response.security_state.clone(),
+                    signed_public_key: Some(enrollment_response.signed_public_key.clone()),
```

Comment:
Clones also not needed here
```suggestion
                    private_key: enrollment_response.private_key,
                    signing_key: enrollment_response.signing_key,
                    security_state: enrollment_response.security_state,
                    signed_public_key: Some(enrollment_response.signed_public_key),
```